examples/comfyui/dev/run_workflow.py

```
@@ -0,0 +1,70 @@
+        if status.get("completed") or record.get("outputs"):
+            break
+    time.sleep(1)
+else:
+    print("timed out polling /history")
+    sys.exit(3)
+
+# 3. extract images from outputs
+outputs = h[prompt_id]["outputs"]
+print("outputs nodes:", list(outputs))
+saved = []
+for node_id, payload in outputs.items():
```

Ruff (0.15.12)

[warning] Loop control variable `node_id` not used within loop body

Rename unused `node_id` to `_node_id`

(B007)